### DISCUSSION TITLE
[Bug] Conda: Several Bugs

### DISCUSSION THREAD

**Discussion by 5675sp** [h2oai/wave#859] [General]
In the wave [documentation](https://wave.h2o.ai/docs/tutorial-hello), the following is instructed if you are using  Conda as your package manager:

```conda install -c h2oai h2o_wave```

The above command raises the following problem: 

```unable to install``` 

I thought maybe the problem was because, according to the conda [website](https://anaconda.org/h2oai/h2o-wave) the command needs to have "-" instead of a "_":

Like this: ```conda install -c h2oai h2o-wave```

That didn't change anything and generated the following issue: 

```package is not compatible``` 

Instead of using the command line above as instructed in the documentation, I decided to do the following: 

```pip install h2o_wave```

After a successful install, I followed to copy the code example on the coda website. This example: 

```
from h2o_wave import site, ui

page = site

page['example'] = ui.markdown_card(
  box='1 1 2 2',
  title='Hello World!',
  content='And now for something completely different.',
)

page.save()`
```

The above code generated the following bug:

```
Traceback (most recent call last):
  File "/Users/sperez/Desktop/aquarium/hello.py", line 5, in <module>
    page['example'] = ui.markdown_card(
AttributeError: __setitem__
```

Wit the above in mind, I replaced the example with something like this: 

```
from h2o_wave import site, ui

page = site['/hello']

page['quote'] = ui.markdown_card(
    box='1 1 2 2',
    title='Hello World',
    content='"The Internet? Is that thing still around?" - *Homer Simpson*',
)

page.save()
```
 
And the above worked.

**Comment by lo5** [comment]
The first example has a bug on line 3: `page = site`.

We don't have open bugs related to the Conda package. What's your environment like?

**Comment by SilentRhetoric** [comment]
Confirming 5675sp's issue with Conda installation and Conda package page.

MacOS 11.4 on Mac mini (M1, 2020)
Conda 4.10.1

```conda install -c h2oai h2o-wave```

Produces the following output:

```Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: | 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                             

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions```
